scripts/interfaces/parameters.ts

```diff
@@ -53,10 +56,13 @@ export interface PortfolioExtraArgs {
 
 export interface StrategyArgs {
   depositToken: string
-  depositFee: Fee
-  withdrawalFee: Fee
-  performanceFee: Fee
-  feeReceiver: FeeReceiver
+  feeArgs: {
```

Comment:
Declaring interface FeeArgs is correspondent to FeeUpgradeable and it also reduces redundancy

Reply:
Sorry I don't get this, I found no 'FeeUpgradeable' in the typescript code.